### ISSUE TITLE
How can I use Modeller to create a prmtop and restart file for a solvated small molecule?

### ISSUE THREAD

**Issue by dim-99** [openmm/openmm#4721]
Hi, 

Is there a direct way to write an amber prmtop and a restart file using modeller? so far I have solvated the system by doing the following, 

modeller = app.Modeller(ligand_topology,to_openmm(ligand_positions))
system_generator = SystemGenerator(
forcefields=["openff-1.0.0.offxml","tip3p.xml"])
modeller.addSolvent(system_generator.forcefield,model='tip3p')

thank you !

**Comment by peastman** [comment]
No, sorry.  It can read Amber files, but not write them.  You might be able to do it with parmed.
